eZ/Publish/Core/Search/Solr/Content/Gateway.php

```diff
@@ -55,6 +55,8 @@
     abstract public function purgeIndex();
 
     /**
+     * @todo workshop: chenge to empty commit request, do not commit by default
```

Comment:
Changed in https://github.com/ezsystems/ezplatform-solr-search-engine/commit/6fadca9ec5b2935cf2983155b010f0f8a250114e